docs/USAGE-GUIDE.md

```diff
@@ -491,14 +491,27 @@ int s2n_config_free(struct s2n_config *config);
 
 **s2n_config_free** frees the memory associated with an **s2n_config** object.
 
+### s2n\_config\_set\_security\_policy
+
+```c
+int s2n_config_set_security_policy(struct s2n_config *config,
+                                      const struct s2n_security_policy *policy);
+```
+
+**s2n_config_set_security_policy** S2N makes opinionated decisions about which ciphers,
+key encapsualtion mechanisms, signature algorithms, and curves can be used together. These
+combinations are called *security policies*. This function is simply a wrapper for
+*s2n_config_set_cipher_preferences*. The name better describes the behavior behind the scenes.
```

Comment:
If _set_security_policy is the preferred API, shouldn't its documentation entry include everything that's currently in the entry for _set_cipher_preferences? Then the entry for _set_cipher_preferences can explain the relationship between the two APIs / why its deprecated. It seems wrong to refer customers to the documentation for an API marked "deprecated."

Reply:
Created an issue to track this. It looks like the policy names match the cipher preference names, but we should verify.
https://github.com/awslabs/s2n/issues/2379

Reply:
I think the discussion today influences this.

If we're not actually deprecating _set_cipher_preferences yet and expect most customers to keep using it, than we can leave its documentation as-is. But maybe we should either remove _set_security_policy from the documentation until we finish the security policy builder, or indicate in its documentation which policies are available from it?

Reply:
I agree, I've removed it from the documentation. I think it will make more sense to document the calls along with their intended usage, once that is built.